Lib/test/test_regrtest.py

```diff
@@ -824,5 +828,53 @@ def test_crashed(self):
                                   randomize=True)
 
 
+    def parse_methods(self, output):
+        regex = re.compile("^(test[^ ]+).*ok$", flags=re.MULTILINE)
+        return [match.group(1) for match in regex.finditer(output)]
+
+    def test_matchfile(self):
```

Comment:
Oh, it would be enough to add a test only in `ParseArgsTestCase`! But more comprehensive test is good.